test/e2e/checkpoint_test.go

```diff
@@ -1110,6 +1110,87 @@ var _ = Describe("Podman checkpoint", func() {
 		"net,uts",
 		"uts,pid",
 	}
+
+	It("podman checkpoint and restore container into Pod", func() {
```

Comment:
Can you just add the podName to one of the existing pod restore tests? These tests are very slow so we should combine many things if possible.

Reply:
Thank you @Luap99! I've updated the pull request to run half of the existing pod restore tests with Pod name and other half with Pod ID.